packages/web-shared/utils/getContentFromURL.js

```diff
@@ -19,58 +15,35 @@ function getContentFromURL(url) {
   const { type, randomId } = parseSlugToIdAndType(url);
 
   switch (type) {
-    case 'EventContentItem':
+    // case 'EventContentItem':
```

Comment:
@linb1 you can just delete this instead of commenting it out.

Reply:
@jordanwade whoops, forgot to delete that. I also deleted a couple other instances of it 👍